model/remotestore/restore/operations.smithy

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+@unstable
+@externalDocumentation(
+    "OpenSearch Documentation": "https://opensearch.org/docs/latest/opensearch/remote/#restoring-from-a-backup"
+)
+
+@suppress(["HttpUriConflict"])
+@http(method: "POST", uri: "/_remotestore/_restore")
+@documentation("Restore one or more indices from a remote backup.")
+operation PostRemoteStoreRestore{
```

Comment:
Are the names of the operation and structures inline with what we would want in the generated code? Or do they need to be changed?

Reply:
They're named to match the naming scheme of the rest of the specs in the repo. Naming (or encoding friendly names as metadata similar to #73) will likely be a bigger issue, but hard to solve in advance until we get to the point of actually generating parts of a client from these.